packages/vite/src/node/server/index.ts

```diff
@@ -97,6 +97,10 @@ export interface ServerOptions extends CommonServerOptions {
    * Configure HMR-specific options (port, host, path & protocol)
    */
   hmr?: HmrOptions | boolean
+  /**
+   * Do not start the websocket connection.
+   */
+  ws?: false
```

Comment:
Given that we are moving to from `ws` to `hot`, I wonder if we should deprecate `hmr` and make `hot` false avoid starting the WebSocket.

I'm ok merging this PR with this option as `@internal` if we want to unblock Vitest here.